content/en/docs/reference/config/analysis/ist0101/index.md

```diff
@@ -0,0 +1,50 @@
+---
+title: ReferencedResourceNotFound
+layout: analysis-message
+---
+
+This message occurs when an Istio resource references another resource that does
+not exist. This will lead to errors when Istio tries to look up the referenced
+resource but cannot find it.
+
+For example, you receive this error:
+
+{{< text yaml >}}
+Error [IST0101] (VirtualService httpbin.default) Referenced gateway not found: "httpbin-gateway-bogus"
```

Comment:
I think it would be worth providing a general description of the error message format:

<number> <affected resource> <error>

In particular, pointing out that the affected resource is what the user is expected to lookup in the cluster and is where a fix should be done. We could have a boilerplate section on each error message page to describe this, or we could put it on a centralized page somewhere.

Reply:
fixed